functions.js

```diff
@@ -183,7 +218,10 @@ Output:
 */
 
 export function getChevyCars(arr) {
-    return [];
+    const chevyArr = arr
+        .filter(car => car.type === 'car')
+        .filter(car => car.make === 'chevy');
+    return chevyArr;
```

Comment:
Nice job with this!  Remember your comparison operators.  You could make these comparisons in the same filter call if you wanted to: `arr.filter(car => car.type === 'car' && car.make === 'chevy');`

Reply:
Yes, that totally makes it more clean.